src/objective-c/manual_tests/GrpcIosTestUITests/GrpcIosTestUITests.m

```diff
@@ -71,41 +99,95 @@ - (void)setAirplaneMode:(BOOL)to {
   [settingsApp activate];
   XCUIElement *mySwitch = settingsApp.tables.element.cells.switches[@"Airplane Mode"];
   BOOL from = [(NSString *)mySwitch.value boolValue];
+  NSLog(@"Setting airplane from: %d to: %d", from, to);
   if (from != to) {
     [mySwitch tap];
     // wait for gRPC to detect the change
-    sleep(10);
+    [NSThread sleepForTimeInterval:20];
```

Comment:
It's pretty much the same as using `sleep`. I just changed it here for consistency because I had used `NSThread sleepForTimeInterval` in other places.
Regarding the sleep time, what do you think is a reasonable amount of time to wait after flapping the network? We just need to wait long enough for the network state change to be detected by gRPC.